### PR TITLE
Update google-cloud-bigquerystorage dependency to 2.1.2 in branch-0.22

### PR DESCRIPTION
**Pull request by esert-g**
This required me to update grpcVersion as well.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -135,7 +135,7 @@ lazy val connector = (project in file("connector"))
 
       // Keep com.google.cloud dependencies in sync
       "com.google.cloud" % "google-cloud-bigquery" % "1.131.1",
```
Comment (davidrabinowitz) [inline build.sbt:137]: Shouldn't google-cloud-bigquery be upgraded to 2.1.7 or is it an unrelated change?

Comment (esert-g) [reply build.sbt:137]: google-cloud-bigquerystorage update to 2.1.2 is only necessary for handling some new RESOURCE_EXHAUSTED errors that may be sent from the bq storage read api calls. I don't know why we would need to update google-cloud-bigquery for that reason.
